lib/seahorn/HornCex.cc

```diff
@@ -95,7 +95,8 @@ using namespace llvm;
 namespace seahorn {
 
 template <typename O> class SvCompCex;
-static void dumpSvCompCex(BmcTrace &trace, std::string CexFile);
+static void dumpSvCompCex(BmcTrace<BmcEngine, ZModel_ref> &trace,
```

Comment:
Add a type for `BmcTrace<BmcEngine, ZModel_ref>` and use throughout. For example, call it `ZBmcTrace`